setup.py

```diff
@@ -20,12 +20,14 @@
         'Stanislav Kudriashev',
         'Raoul Snyman',
         'Corey Goldberg',
+        'Harald Nordgren',
     ]),
     url='https://github.com/mahmoudimus/nose-timer',
     packages=['nosetimer', ],
     install_requires=[
         'nose',
         'termcolor',
+        'ordereddict',
```

Comment:
Why is this always required? Shouldn't only be for fairly old Pythons (e.g. before 2.7).